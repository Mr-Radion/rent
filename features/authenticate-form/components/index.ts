export { AuthenticationFormModal } from './auth-form-modal-test';
export { LoginForm } from './login-form';
export { RegisterForm } from './register-form';
