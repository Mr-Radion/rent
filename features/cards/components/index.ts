export { AdsReview } from './card-review';
export { CardsList } from './card-list';
