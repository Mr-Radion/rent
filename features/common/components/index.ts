export { Header } from './header';
export { TopNav } from './navbar';
export { Footer } from './footer';
export { MainNav } from './navbar';
export { CenterContent } from './main-center';
export { UserMenu } from './user-menu';
