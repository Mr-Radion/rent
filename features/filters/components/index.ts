export { FilterTypeProperty } from './filter-type-property';
export { FilterPrice } from './filter-price';
export { FilterBedrooms } from './filter-bedrooms';
export { FilterArea } from './filter-area';
export { FilterLocation } from './filter-location';
