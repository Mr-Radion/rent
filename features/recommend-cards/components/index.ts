export { RecommendedAdsItem } from './recommend-cards';
export type { RecommendedAdsItemProps } from './recommend-cards';
export { LoadingRecommendedBlock } from './recommended-card-loader';
