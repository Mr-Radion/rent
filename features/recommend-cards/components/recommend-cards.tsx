import styled from 'styled-components';
import { MapMarker } from '../../../ui';
import { RecommendedAdsState } from '../ducks';

const CartBlock = styled.div`
  flex-basis: calc((100% / 4) - 3.83rem);
  @media (max-width: 1366px) {
    flex-basis: calc((100% / 4) - 2rem);
  }
  @media (max-width: 1024px) {
    flex-basis: calc((100% / 3) - 1rem);
  }
  @media (max-width: 768px) {
    flex-basis: calc((100% / 3, 3) - 1rem);
  }
  @media (max-width: 600px) {
    flex-basis: calc((100% / 2) - 1rem);
    height: 300px;
  }
  @media (max-width: 500px) {
    flex-basis: calc(100% - 1rem);
  }
`;

const CartWrapper = styled.div`
  display: flex;
  flex-direction: column;
  // width: 19.81rem;
  margin-bottom: 20px;
  box-shadow: 0px 2px 8px rgb(119 204 209 / 25%);
  border-radius: 4px;
`;

const CartContent = styled.div`
  padding: 8.25px 10px 8px;
`;

const CartAdress = styled.div`
  padding: 0 13px 11.1px;
  font-weight: 900;
  font-size: 14px;
  line-height: 22px;
  // opacity: 0.8;
  color: #4b4b4b;
  letter-spacing: 0.5px;
  > span {
    padding-right: 7px;
  }
`;

const PhotoBox = styled.div`
  display: flex;
  align-items: center;
  justify-content: center;
  height: 184px;
  background-color: #c4c4c4;
  position: relative;
  img {
    position: absolute;
    min-width: 100%;
    max-width: 100%;
    min-height: 100%;
    max-height: 100%;
    object-fit: cover;
  }
`;

const PriceBlock = styled.div`
  font-style: normal;
  font-weight: 900;
  font-size: 24px;
  line-height: 38px;
  letter-spacing: 0.5px;
`;

const TextBlock = styled.div`
  font-size: 18px;
  line-height: 26px;
  // opacity: 0.8;
`;

export interface RecommendedAdsItemProps {
  cardItems: RecommendedAdsState[];
}

export const RecommendedAdsItem = ({ cardItems }: RecommendedAdsItemProps): JSX.Element => (
  <>
    {cardItems &&
      cardItems.map(card => (
        <CartBlock key={card.id}>
          <CartWrapper>
            <PhotoBox>
              <img src={card.image} alt={card.name} loading="lazy" decoding="async" />;
            </PhotoBox>
            <CartContent>
              <PriceBlock>
                {card.price}
                <span>â‚¬</span>
              </PriceBlock>
              <TextBlock>
                {card.rooms}
                <span> rooms, </span>
                123
                <span> m2</span>
              </TextBlock>
            </CartContent>
            <CartAdress>
              <span>
                <MapMarker width="14" height="17" />
              </span>
              {card.city}
            </CartAdress>
          </CartWrapper>
        </CartBlock>
      ))}
  </>
);
