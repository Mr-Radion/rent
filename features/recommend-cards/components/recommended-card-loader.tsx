import React from 'react';
import styled from 'styled-components';
import ContentLoader from 'react-content-loader';

const CartWrapper = styled.div`
  display: flex;
  flex-direction: column;
  width: 19.81rem;
  margin-bottom: 20px;
  box-shadow: 0px 2px 8px rgb(119 204 209 / 25%);
  border-radius: 4px;
`;

export const LoadingRecommendedBlock: React.FC<any> = props => (
  <CartWrapper>
    <ContentLoader
      speed={2}
      width={316.95}
      height={297.15}
      viewBox="0 0 316.950 297.150"
      backgroundColor="#f3f3f3"
      foregroundColor="#ecebeb"
      {...props}
    >
      <rect x="10" y="192" rx="0" ry="0" width="92" height="36" />
      <rect x="0" y="0" rx="4" ry="4" width="317" height="184" />
      <rect x="10" y="274" rx="0" ry="0" width="97" height="21" />
      <rect x="10" y="229" rx="0" ry="0" width="142" height="37" />
    </ContentLoader>
  </CartWrapper>
);
