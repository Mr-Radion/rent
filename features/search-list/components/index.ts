export { SearchCart, SearchBar } from './search-cart';
