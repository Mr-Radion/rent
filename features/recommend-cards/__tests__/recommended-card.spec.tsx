import React from 'react';
import { render } from '../../../test/test-utils';
import { RecommendedAdsItem } from '../components';
import data from '../../../lib/mock-data/db.json';

describe('RecommendedAdsItem', () => {
  it('matches snapshot', () => {
    const { asFragment } = render(<RecommendedAdsItem cardItems={data.adsRecommended} />, {});
    expect(asFragment()).toMatchSnapshot();
  });
});
