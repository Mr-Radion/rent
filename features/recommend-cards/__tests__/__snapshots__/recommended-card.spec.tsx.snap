// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RecommendedAdsItem matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="recommend-cards__CartBlock-sc-5yu6d2-0 iXYyft"
  >
    <div
      class="recommend-cards__CartWrapper-sc-5yu6d2-1 dwAudU"
    >
      <a
        href="/myad/1"
        target="_blank"
      >
        <div
          class="photobox__PhotoBoxStyled-smpouz-0 jUHvzi"
          height="184px"
        >
          <img
            alt="string"
            decoding="async"
            loading="lazy"
            src="https://unsplash.com/photos/2d4lAQAlbDA"
          />
          ;
        </div>
      </a>
      <div
        class="recommend-cards__CartContent-sc-5yu6d2-2 txYE"
      >
        <div
          class="recommend-cards__PriceBlock-sc-5yu6d2-4 knJDVs"
        >
          4000
          <span>
            €
          </span>
        </div>
        <div
          class="recommend-cards__TextBlock-sc-5yu6d2-5 kpFVYA"
        >
          3
          <span>
             rooms, 
          </span>
          123
          <span>
             m2
          </span>
        </div>
      </div>
      <div
        class="recommend-cards__CartAdress-sc-5yu6d2-3 kZpHEI"
      >
        <span>
          <svg
            class="map-marker__StyledMapMarker-tw1h2b-0 gPGwoF"
            fill="none"
            height="17"
            viewBox="0 0 14 17"
            width="14"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              clip-rule="evenodd"
              d="M2.05002 2.04999C3.36284 0.73717 5.14341 -0.000366211 7.00002 -0.000366211C8.85663 -0.000366211 10.6372 0.73717 11.95 2.04999C13.2628 3.36281 14.0004 5.14338 14.0004 6.99999C14.0004 8.8566 13.2628 10.6372 11.95 11.95L7.00002 16.9L2.05002 11.95C1.39993 11.3 0.884251 10.5283 0.532423 9.67894C0.180596 8.82961 -0.000488281 7.9193 -0.000488281 6.99999C-0.000488281 6.08068 0.180596 5.17037 0.532423 4.32104C0.884251 3.47172 1.39993 2.70001 2.05002 2.04999ZM7.00002 8.99999C7.53045 8.99999 8.03916 8.78928 8.41423 8.4142C8.7893 8.03913 9.00002 7.53042 9.00002 6.99999C9.00002 6.46956 8.7893 5.96085 8.41423 5.58578C8.03916 5.2107 7.53045 4.99999 7.00002 4.99999C6.46958 4.99999 5.96088 5.2107 5.5858 5.58578C5.21073 5.96085 5.00002 6.46956 5.00002 6.99999C5.00002 7.53042 5.21073 8.03913 5.5858 8.4142C5.96088 8.78928 6.46958 8.99999 7.00002 8.99999Z"
              fill="#1995AD"
              fill-rule="evenodd"
            />
          </svg>
        </span>
        string
      </div>
    </div>
  </div>
  <div
    class="recommend-cards__CartBlock-sc-5yu6d2-0 iXYyft"
  >
    <div
      class="recommend-cards__CartWrapper-sc-5yu6d2-1 dwAudU"
    >
      <a
        href="/myad/2"
        target="_blank"
      >
        <div
          class="photobox__PhotoBoxStyled-smpouz-0 jUHvzi"
          height="184px"
        >
          <img
            alt="string"
            decoding="async"
            loading="lazy"
            src="https://unsplash.com/photos/2d4lAQAlbDA"
          />
          ;
        </div>
      </a>
      <div
        class="recommend-cards__CartContent-sc-5yu6d2-2 txYE"
      >
        <div
          class="recommend-cards__PriceBlock-sc-5yu6d2-4 knJDVs"
        >
          4000
          <span>
            €
          </span>
        </div>
        <div
          class="recommend-cards__TextBlock-sc-5yu6d2-5 kpFVYA"
        >
          3
          <span>
             rooms, 
          </span>
          123
          <span>
             m2
          </span>
        </div>
      </div>
      <div
        class="recommend-cards__CartAdress-sc-5yu6d2-3 kZpHEI"
      >
        <span>
          <svg
            class="map-marker__StyledMapMarker-tw1h2b-0 gPGwoF"
            fill="none"
            height="17"
            viewBox="0 0 14 17"
            width="14"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              clip-rule="evenodd"
              d="M2.05002 2.04999C3.36284 0.73717 5.14341 -0.000366211 7.00002 -0.000366211C8.85663 -0.000366211 10.6372 0.73717 11.95 2.04999C13.2628 3.36281 14.0004 5.14338 14.0004 6.99999C14.0004 8.8566 13.2628 10.6372 11.95 11.95L7.00002 16.9L2.05002 11.95C1.39993 11.3 0.884251 10.5283 0.532423 9.67894C0.180596 8.82961 -0.000488281 7.9193 -0.000488281 6.99999C-0.000488281 6.08068 0.180596 5.17037 0.532423 4.32104C0.884251 3.47172 1.39993 2.70001 2.05002 2.04999ZM7.00002 8.99999C7.53045 8.99999 8.03916 8.78928 8.41423 8.4142C8.7893 8.03913 9.00002 7.53042 9.00002 6.99999C9.00002 6.46956 8.7893 5.96085 8.41423 5.58578C8.03916 5.2107 7.53045 4.99999 7.00002 4.99999C6.46958 4.99999 5.96088 5.2107 5.5858 5.58578C5.21073 5.96085 5.00002 6.46956 5.00002 6.99999C5.00002 7.53042 5.21073 8.03913 5.5858 8.4142C5.96088 8.78928 6.46958 8.99999 7.00002 8.99999Z"
              fill="#1995AD"
              fill-rule="evenodd"
            />
          </svg>
        </span>
        string
      </div>
    </div>
  </div>
  <div
    class="recommend-cards__CartBlock-sc-5yu6d2-0 iXYyft"
  >
    <div
      class="recommend-cards__CartWrapper-sc-5yu6d2-1 dwAudU"
    >
      <a
        href="/myad/3"
        target="_blank"
      >
        <div
          class="photobox__PhotoBoxStyled-smpouz-0 jUHvzi"
          height="184px"
        >
          <img
            alt="string"
            decoding="async"
            loading="lazy"
            src="https://unsplash.com/photos/2d4lAQAlbDA"
          />
          ;
        </div>
      </a>
      <div
        class="recommend-cards__CartContent-sc-5yu6d2-2 txYE"
      >
        <div
          class="recommend-cards__PriceBlock-sc-5yu6d2-4 knJDVs"
        >
          4000
          <span>
            €
          </span>
        </div>
        <div
          class="recommend-cards__TextBlock-sc-5yu6d2-5 kpFVYA"
        >
          3
          <span>
             rooms, 
          </span>
          123
          <span>
             m2
          </span>
        </div>
      </div>
      <div
        class="recommend-cards__CartAdress-sc-5yu6d2-3 kZpHEI"
      >
        <span>
          <svg
            class="map-marker__StyledMapMarker-tw1h2b-0 gPGwoF"
            fill="none"
            height="17"
            viewBox="0 0 14 17"
            width="14"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              clip-rule="evenodd"
              d="M2.05002 2.04999C3.36284 0.73717 5.14341 -0.000366211 7.00002 -0.000366211C8.85663 -0.000366211 10.6372 0.73717 11.95 2.04999C13.2628 3.36281 14.0004 5.14338 14.0004 6.99999C14.0004 8.8566 13.2628 10.6372 11.95 11.95L7.00002 16.9L2.05002 11.95C1.39993 11.3 0.884251 10.5283 0.532423 9.67894C0.180596 8.82961 -0.000488281 7.9193 -0.000488281 6.99999C-0.000488281 6.08068 0.180596 5.17037 0.532423 4.32104C0.884251 3.47172 1.39993 2.70001 2.05002 2.04999ZM7.00002 8.99999C7.53045 8.99999 8.03916 8.78928 8.41423 8.4142C8.7893 8.03913 9.00002 7.53042 9.00002 6.99999C9.00002 6.46956 8.7893 5.96085 8.41423 5.58578C8.03916 5.2107 7.53045 4.99999 7.00002 4.99999C6.46958 4.99999 5.96088 5.2107 5.5858 5.58578C5.21073 5.96085 5.00002 6.46956 5.00002 6.99999C5.00002 7.53042 5.21073 8.03913 5.5858 8.4142C5.96088 8.78928 6.46958 8.99999 7.00002 8.99999Z"
              fill="#1995AD"
              fill-rule="evenodd"
            />
          </svg>
        </span>
        string
      </div>
    </div>
  </div>
  <div
    class="recommend-cards__CartBlock-sc-5yu6d2-0 iXYyft"
  >
    <div
      class="recommend-cards__CartWrapper-sc-5yu6d2-1 dwAudU"
    >
      <a
        href="/myad/4"
        target="_blank"
      >
        <div
          class="photobox__PhotoBoxStyled-smpouz-0 jUHvzi"
          height="184px"
        >
          <img
            alt="string"
            decoding="async"
            loading="lazy"
            src="https://unsplash.com/photos/2d4lAQAlbDA"
          />
          ;
        </div>
      </a>
      <div
        class="recommend-cards__CartContent-sc-5yu6d2-2 txYE"
      >
        <div
          class="recommend-cards__PriceBlock-sc-5yu6d2-4 knJDVs"
        >
          4000
          <span>
            €
          </span>
        </div>
        <div
          class="recommend-cards__TextBlock-sc-5yu6d2-5 kpFVYA"
        >
          3
          <span>
             rooms, 
          </span>
          123
          <span>
             m2
          </span>
        </div>
      </div>
      <div
        class="recommend-cards__CartAdress-sc-5yu6d2-3 kZpHEI"
      >
        <span>
          <svg
            class="map-marker__StyledMapMarker-tw1h2b-0 gPGwoF"
            fill="none"
            height="17"
            viewBox="0 0 14 17"
            width="14"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              clip-rule="evenodd"
              d="M2.05002 2.04999C3.36284 0.73717 5.14341 -0.000366211 7.00002 -0.000366211C8.85663 -0.000366211 10.6372 0.73717 11.95 2.04999C13.2628 3.36281 14.0004 5.14338 14.0004 6.99999C14.0004 8.8566 13.2628 10.6372 11.95 11.95L7.00002 16.9L2.05002 11.95C1.39993 11.3 0.884251 10.5283 0.532423 9.67894C0.180596 8.82961 -0.000488281 7.9193 -0.000488281 6.99999C-0.000488281 6.08068 0.180596 5.17037 0.532423 4.32104C0.884251 3.47172 1.39993 2.70001 2.05002 2.04999ZM7.00002 8.99999C7.53045 8.99999 8.03916 8.78928 8.41423 8.4142C8.7893 8.03913 9.00002 7.53042 9.00002 6.99999C9.00002 6.46956 8.7893 5.96085 8.41423 5.58578C8.03916 5.2107 7.53045 4.99999 7.00002 4.99999C6.46958 4.99999 5.96088 5.2107 5.5858 5.58578C5.21073 5.96085 5.00002 6.46956 5.00002 6.99999C5.00002 7.53042 5.21073 8.03913 5.5858 8.4142C5.96088 8.78928 6.46958 8.99999 7.00002 8.99999Z"
              fill="#1995AD"
              fill-rule="evenodd"
            />
          </svg>
        </span>
        string
      </div>
    </div>
  </div>
  <div
    class="recommend-cards__CartBlock-sc-5yu6d2-0 iXYyft"
  >
    <div
      class="recommend-cards__CartWrapper-sc-5yu6d2-1 dwAudU"
    >
      <a
        href="/myad/5"
        target="_blank"
      >
        <div
          class="photobox__PhotoBoxStyled-smpouz-0 jUHvzi"
          height="184px"
        >
          <img
            alt="string"
            decoding="async"
            loading="lazy"
            src="https://unsplash.com/photos/2d4lAQAlbDA"
          />
          ;
        </div>
      </a>
      <div
        class="recommend-cards__CartContent-sc-5yu6d2-2 txYE"
      >
        <div
          class="recommend-cards__PriceBlock-sc-5yu6d2-4 knJDVs"
        >
          4000
          <span>
            €
          </span>
        </div>
        <div
          class="recommend-cards__TextBlock-sc-5yu6d2-5 kpFVYA"
        >
          3
          <span>
             rooms, 
          </span>
          123
          <span>
             m2
          </span>
        </div>
      </div>
      <div
        class="recommend-cards__CartAdress-sc-5yu6d2-3 kZpHEI"
      >
        <span>
          <svg
            class="map-marker__StyledMapMarker-tw1h2b-0 gPGwoF"
            fill="none"
            height="17"
            viewBox="0 0 14 17"
            width="14"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              clip-rule="evenodd"
              d="M2.05002 2.04999C3.36284 0.73717 5.14341 -0.000366211 7.00002 -0.000366211C8.85663 -0.000366211 10.6372 0.73717 11.95 2.04999C13.2628 3.36281 14.0004 5.14338 14.0004 6.99999C14.0004 8.8566 13.2628 10.6372 11.95 11.95L7.00002 16.9L2.05002 11.95C1.39993 11.3 0.884251 10.5283 0.532423 9.67894C0.180596 8.82961 -0.000488281 7.9193 -0.000488281 6.99999C-0.000488281 6.08068 0.180596 5.17037 0.532423 4.32104C0.884251 3.47172 1.39993 2.70001 2.05002 2.04999ZM7.00002 8.99999C7.53045 8.99999 8.03916 8.78928 8.41423 8.4142C8.7893 8.03913 9.00002 7.53042 9.00002 6.99999C9.00002 6.46956 8.7893 5.96085 8.41423 5.58578C8.03916 5.2107 7.53045 4.99999 7.00002 4.99999C6.46958 4.99999 5.96088 5.2107 5.5858 5.58578C5.21073 5.96085 5.00002 6.46956 5.00002 6.99999C5.00002 7.53042 5.21073 8.03913 5.5858 8.4142C5.96088 8.78928 6.46958 8.99999 7.00002 8.99999Z"
              fill="#1995AD"
              fill-rule="evenodd"
            />
          </svg>
        </span>
        string
      </div>
    </div>
  </div>
  <div
    class="recommend-cards__CartBlock-sc-5yu6d2-0 iXYyft"
  >
    <div
      class="recommend-cards__CartWrapper-sc-5yu6d2-1 dwAudU"
    >
      <a
        href="/myad/6"
        target="_blank"
      >
        <div
          class="photobox__PhotoBoxStyled-smpouz-0 jUHvzi"
          height="184px"
        >
          <img
            alt="string"
            decoding="async"
            loading="lazy"
            src="https://unsplash.com/photos/2d4lAQAlbDA"
          />
          ;
        </div>
      </a>
      <div
        class="recommend-cards__CartContent-sc-5yu6d2-2 txYE"
      >
        <div
          class="recommend-cards__PriceBlock-sc-5yu6d2-4 knJDVs"
        >
          4000
          <span>
            €
          </span>
        </div>
        <div
          class="recommend-cards__TextBlock-sc-5yu6d2-5 kpFVYA"
        >
          3
          <span>
             rooms, 
          </span>
          123
          <span>
             m2
          </span>
        </div>
      </div>
      <div
        class="recommend-cards__CartAdress-sc-5yu6d2-3 kZpHEI"
      >
        <span>
          <svg
            class="map-marker__StyledMapMarker-tw1h2b-0 gPGwoF"
            fill="none"
            height="17"
            viewBox="0 0 14 17"
            width="14"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              clip-rule="evenodd"
              d="M2.05002 2.04999C3.36284 0.73717 5.14341 -0.000366211 7.00002 -0.000366211C8.85663 -0.000366211 10.6372 0.73717 11.95 2.04999C13.2628 3.36281 14.0004 5.14338 14.0004 6.99999C14.0004 8.8566 13.2628 10.6372 11.95 11.95L7.00002 16.9L2.05002 11.95C1.39993 11.3 0.884251 10.5283 0.532423 9.67894C0.180596 8.82961 -0.000488281 7.9193 -0.000488281 6.99999C-0.000488281 6.08068 0.180596 5.17037 0.532423 4.32104C0.884251 3.47172 1.39993 2.70001 2.05002 2.04999ZM7.00002 8.99999C7.53045 8.99999 8.03916 8.78928 8.41423 8.4142C8.7893 8.03913 9.00002 7.53042 9.00002 6.99999C9.00002 6.46956 8.7893 5.96085 8.41423 5.58578C8.03916 5.2107 7.53045 4.99999 7.00002 4.99999C6.46958 4.99999 5.96088 5.2107 5.5858 5.58578C5.21073 5.96085 5.00002 6.46956 5.00002 6.99999C5.00002 7.53042 5.21073 8.03913 5.5858 8.4142C5.96088 8.78928 6.46958 8.99999 7.00002 8.99999Z"
              fill="#1995AD"
              fill-rule="evenodd"
            />
          </svg>
        </span>
        string
      </div>
    </div>
  </div>
  <div
    class="recommend-cards__CartBlock-sc-5yu6d2-0 iXYyft"
  >
    <div
      class="recommend-cards__CartWrapper-sc-5yu6d2-1 dwAudU"
    >
      <a
        href="/myad/7"
        target="_blank"
      >
        <div
          class="photobox__PhotoBoxStyled-smpouz-0 jUHvzi"
          height="184px"
        >
          <img
            alt="string"
            decoding="async"
            loading="lazy"
            src="https://unsplash.com/photos/2d4lAQAlbDA"
          />
          ;
        </div>
      </a>
      <div
        class="recommend-cards__CartContent-sc-5yu6d2-2 txYE"
      >
        <div
          class="recommend-cards__PriceBlock-sc-5yu6d2-4 knJDVs"
        >
          4000
          <span>
            €
          </span>
        </div>
        <div
          class="recommend-cards__TextBlock-sc-5yu6d2-5 kpFVYA"
        >
          3
          <span>
             rooms, 
          </span>
          123
          <span>
             m2
          </span>
        </div>
      </div>
      <div
        class="recommend-cards__CartAdress-sc-5yu6d2-3 kZpHEI"
      >
        <span>
          <svg
            class="map-marker__StyledMapMarker-tw1h2b-0 gPGwoF"
            fill="none"
            height="17"
            viewBox="0 0 14 17"
            width="14"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              clip-rule="evenodd"
              d="M2.05002 2.04999C3.36284 0.73717 5.14341 -0.000366211 7.00002 -0.000366211C8.85663 -0.000366211 10.6372 0.73717 11.95 2.04999C13.2628 3.36281 14.0004 5.14338 14.0004 6.99999C14.0004 8.8566 13.2628 10.6372 11.95 11.95L7.00002 16.9L2.05002 11.95C1.39993 11.3 0.884251 10.5283 0.532423 9.67894C0.180596 8.82961 -0.000488281 7.9193 -0.000488281 6.99999C-0.000488281 6.08068 0.180596 5.17037 0.532423 4.32104C0.884251 3.47172 1.39993 2.70001 2.05002 2.04999ZM7.00002 8.99999C7.53045 8.99999 8.03916 8.78928 8.41423 8.4142C8.7893 8.03913 9.00002 7.53042 9.00002 6.99999C9.00002 6.46956 8.7893 5.96085 8.41423 5.58578C8.03916 5.2107 7.53045 4.99999 7.00002 4.99999C6.46958 4.99999 5.96088 5.2107 5.5858 5.58578C5.21073 5.96085 5.00002 6.46956 5.00002 6.99999C5.00002 7.53042 5.21073 8.03913 5.5858 8.4142C5.96088 8.78928 6.46958 8.99999 7.00002 8.99999Z"
              fill="#1995AD"
              fill-rule="evenodd"
            />
          </svg>
        </span>
        string
      </div>
    </div>
  </div>
  <div
    class="recommend-cards__CartBlock-sc-5yu6d2-0 iXYyft"
  >
    <div
      class="recommend-cards__CartWrapper-sc-5yu6d2-1 dwAudU"
    >
      <a
        href="/myad/8"
        target="_blank"
      >
        <div
          class="photobox__PhotoBoxStyled-smpouz-0 jUHvzi"
          height="184px"
        >
          <img
            alt="string"
            decoding="async"
            loading="lazy"
            src="https://unsplash.com/photos/2d4lAQAlbDA"
          />
          ;
        </div>
      </a>
      <div
        class="recommend-cards__CartContent-sc-5yu6d2-2 txYE"
      >
        <div
          class="recommend-cards__PriceBlock-sc-5yu6d2-4 knJDVs"
        >
          4000
          <span>
            €
          </span>
        </div>
        <div
          class="recommend-cards__TextBlock-sc-5yu6d2-5 kpFVYA"
        >
          3
          <span>
             rooms, 
          </span>
          123
          <span>
             m2
          </span>
        </div>
      </div>
      <div
        class="recommend-cards__CartAdress-sc-5yu6d2-3 kZpHEI"
      >
        <span>
          <svg
            class="map-marker__StyledMapMarker-tw1h2b-0 gPGwoF"
            fill="none"
            height="17"
            viewBox="0 0 14 17"
            width="14"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              clip-rule="evenodd"
              d="M2.05002 2.04999C3.36284 0.73717 5.14341 -0.000366211 7.00002 -0.000366211C8.85663 -0.000366211 10.6372 0.73717 11.95 2.04999C13.2628 3.36281 14.0004 5.14338 14.0004 6.99999C14.0004 8.8566 13.2628 10.6372 11.95 11.95L7.00002 16.9L2.05002 11.95C1.39993 11.3 0.884251 10.5283 0.532423 9.67894C0.180596 8.82961 -0.000488281 7.9193 -0.000488281 6.99999C-0.000488281 6.08068 0.180596 5.17037 0.532423 4.32104C0.884251 3.47172 1.39993 2.70001 2.05002 2.04999ZM7.00002 8.99999C7.53045 8.99999 8.03916 8.78928 8.41423 8.4142C8.7893 8.03913 9.00002 7.53042 9.00002 6.99999C9.00002 6.46956 8.7893 5.96085 8.41423 5.58578C8.03916 5.2107 7.53045 4.99999 7.00002 4.99999C6.46958 4.99999 5.96088 5.2107 5.5858 5.58578C5.21073 5.96085 5.00002 6.46956 5.00002 6.99999C5.00002 7.53042 5.21073 8.03913 5.5858 8.4142C5.96088 8.78928 6.46958 8.99999 7.00002 8.99999Z"
              fill="#1995AD"
              fill-rule="evenodd"
            />
          </svg>
        </span>
        string
      </div>
    </div>
  </div>
</DocumentFragment>
`;
